docs/setup/figma.md

```diff
@@ -0,0 +1,42 @@
+# Figma
+
+## Setting up
+
+To bridge Figma webhooks with Hookshot, you will need:
+ - A personal access token with admin access to the team you intend to bridge.
+ - A figma account that is on the professional tier, as the free tier does provide webhook access.
+ - Your team ID. You can get this by going to the team page on Figma, and looking for the ID in the url (e.g. 12345 in `https://www.figma.com/files/team/12345/...`)
+
+## Configuration
+
+You can now set some configuration in the bridge `config.yml`
+
+```yaml
+figma:
+  publicUrl: https://example.com/hookshot/
```

Comment:
@Half-Shot this doesn't seem right according to the below description. should be `https://example.com/hookshot/figma/webhook`?

Reply:
It also feels worth pointing out the inconsistence that the other services do not appear to have such specific `/service/webhook` endpoints

Reply:
Ah, the below description is wrong, will fix.